modules/core/04-channel/keeper/packet_test.go

```diff
@@ -236,19 +236,23 @@ func (suite *KeeperTestSuite) TestSendPacket() {
 				sourceChannel = path.EndpointA.ChannelID
 
 				channel := path.EndpointA.GetChannel()
-				channel.FlushStatus = types.FLUSHCOMPLETE
+				channel.State = types.STATE_FLUSHCOMPLETE
```

Comment:
we should rename this testcase to something like `"channel is in FLUSHCOMPLETE state"`

Reply:
yea, and drop the other test case that hardcodes the state to `FLUSHING` since a test that moves it there via the core handlers exists.